lib/BloqadeExpr/Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.1.13"
 [deps]
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 BitBasis = "50ba71b6-fa0f-514d-ae9a-0916efc90dcf"
+BloqadeKrylov = "bd27d05e-4cd1-5e79-84dd-c5d7d508ade2"
```

Comment:
Do we need this dependency?

Reply:
I don't think we need that I will remove it